src/Microsoft.Azure.Functions.Worker.Extensions.Mcp.Placeholder/Worker.Extensions.Mcp.Placeholder.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net10.0</TargetFramework>
+    <AssemblyName>Microsoft.Azure.Functions.Worker.Extensions.Mcp.Placeholder</AssemblyName>
```

Comment:
Looking for real name ideas for this project 

`Microsoft.Azure.Functions.Worker.Extensions.Mcp.Core` makes sense for the original extension project but not this one imo. And I know .Sdk doesnt make sense because this is not an SDK, jusy brings in the MCP SDK. 

Naming is hard :D

Reply:
`Microsoft.Azure.Functions.Worker.Extensions.Mcp.Sdk` is not entirely incorrect. `Microsoft.Azure.Functions.Worker.Extensions.Mcp.SdkIntegration` is perhaps another option worth considering.

The simple way to position this project is that it provides integration with the MCP SDK.

This enables use of the MCP SDK models, APIs, etc., without bringing that dependency into the core extension. In addition to addressing the release quality/lifecycle issue, it leaves the ability to bring integration with other implementations open.

Reply:
So I did originally name it `Microsoft.Azure.Functions.Worker.Extensions.Mcp.Sdk` (before changing to temp placeholder name) but I think @jviau mentioned in standup that it shouldnt be called that because it's not an actual SDK 😂 But it we're all in agreement I am happy to go with `Microsoft.Azure.Functions.Worker.Extensions.Mcp.Sdk`

Reply:
I did? Apologies. Naming is hard. There is no ideal name here.